tools/lkl/cptofs.c

```diff
@@ -493,6 +493,9 @@ int copy_one(const char *src, const char *mpoint, const char *dst)
 		snprintf(dst_path, sizeof(dst_path),  "%s", dst);
 	}
 
+	if (strcmp(src, "/") == 0)
```

Comment:
@copumpkin Good point. I think the equivalent regular expression [./]* should cover all cases. .. already works fine so we don't need consider it. Anything else?